fcs/FSharp.Compiler.Service/FSharp.Compiler.Service.fsproj

```diff
@@ -723,6 +728,9 @@
     <PackageReference Include="System.Buffers" Version="4.5.0" />
     <PackageReference Include="System.Memory" Version="4.5.3" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildVersion)" />
```

Comment:
These dependencies are problematic, see #10303 

Also they aren't listed in the nupkg, see https://www.nuget.org/packages/FSharp.Compiler.Service/38.0.0

cc @baronfel